assets/src/dashboard/app/views/editorSettings/googleAnalytics/index.js

```diff
@@ -158,55 +158,66 @@ function GoogleAnalyticsSettings({
-          <VisuallyHiddenLabel htmlFor="gaTrackingId">
-            {TEXT.ARIA_LABEL}
-          </VisuallyHiddenLabel>
-          <SettingsTextInput
-            aria-label={TEXT.ARIA_LABEL}
-            id="gaTrackingId"
-            value={analyticsId}
-            onChange={handleUpdateId}
-            onKeyDown={handleOnKeyDown}
-            placeholder={TEXT.PLACEHOLDER}
-            hasError={Boolean(inputError)}
-            hint={inputError}
-            disabled={analyticsActive}
-          />
-          <SaveButton
-            type={BUTTON_TYPES.SECONDARY}
-            size={BUTTON_SIZES.SMALL}
-            disabled={disableSaveButton}
-            onClick={handleOnSave}
+      {installed && analyticsActive ? (
+        <div>
+          <TextInputHelperText
+            size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.SMALL}
           >
-            {TEXT.SUBMIT_BUTTON}
-          </SaveButton>
-        </InlineForm>
-        <TextInputHelperText
-          size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.SMALL}
-        >
-          <TranslateWithMarkup
-            mapping={{
-              a: (
-                <InlineLink
-                  href={TEXT.CONTEXT_LINK}
-                  rel="noreferrer"
-                  target="_blank"
-                  size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.SMALL}
-                  as="a"
-                  onClick={onContextClick}
-                />
-              ),
-            }}
+            {TEXT.SITE_KIT_IN_USE}
+          </TextInputHelperText>
+        </div>
+      ) : (
+        <div>
+          <InlineForm>
+            <VisuallyHiddenLabel htmlFor="gaTrackingId">
+              {TEXT.ARIA_LABEL}
+            </VisuallyHiddenLabel>
+            <SettingsTextInput
+              aria-label={TEXT.ARIA_LABEL}
+              id="gaTrackingId"
+              value={analyticsId}
+              onChange={handleUpdateId}
+              onKeyDown={handleOnKeyDown}
+              placeholder={TEXT.PLACEHOLDER}
+              hasError={Boolean(inputError)}
+              hint={inputError}
+              disabled={analyticsActive}
+            />
+            <SaveButton
+              type={BUTTON_TYPES.SECONDARY}
+              size={BUTTON_SIZES.SMALL}
+              disabled={disableSaveButton}
+              onClick={handleOnSave}
+            >
+              {TEXT.SUBMIT_BUTTON}
+            </SaveButton>
+          </InlineForm>
+          <TextInputHelperText
+            size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.SMALL}
           >
-            {TEXT.CONTEXT}
-          </TranslateWithMarkup>
-        </TextInputHelperText>
-      </div>
+            <TranslateWithMarkup
+              mapping={{
+                a: (
+                  <InlineLink
+                    href={TEXT.CONTEXT_LINK}
+                    rel="noreferrer"
+                    target="_blank"
+                    size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.SMALL}
+                    as="a"
```

Comment:
I don't think we need this - should already be an `a` tag
```suggestion
```

Reply:
Fixed in https://github.com/google/web-stories-wp/pull/7622

Reply:
Depending on which PR gets first merged, there might be a conflict and it might slip through.

Let's keep an eye on it.